main.cpp

```diff
@@ -40,24 +42,27 @@ struct Person
 };
 void Person::Foot::stepForward()
 {
-
+    length = 1.0f;
 }
 int Person::Foot::stepSize(int howLong)
 {
     
-    return{};
+    howLong =1;
+    length = 2.0f;
 }
```

Comment:
This function has been declared to return an `int`.  
Does it return an `int`? 
also, fix your spacing around your `=`